src/iOS.Core/Views/FormEntryTableViewCell.cs

```diff
@@ -174,5 +165,74 @@ public void Select()
                 TextField.BecomeFirstResponder();
             }
         }
+
+        public void ConfigureToggleSecureTextCell(bool useSecondaryButton = false)
+        {
+            var button = useSecondaryButton ? SecondButton : Button;
+            button.TitleLabel.Font = UIFont.FromName("bwi-font", 28f);
+            button.SetTitle(BitwardenIcons.Eye, UIControlState.Normal);
+            button.TouchUpInside += (sender, e) =>
+            {
+                TextField.SecureTextEntry = !TextField.SecureTextEntry;
+                button.SetTitle(TextField.SecureTextEntry ? BitwardenIcons.Eye : BitwardenIcons.EyeSlash, UIControlState.Normal);
+            };
+        }
+
+        private void AddButtons(ButtonsConfig buttonsConfig)
+        {
+            Button = new UIButton(UIButtonType.System);
+            Button.TranslatesAutoresizingMaskIntoConstraints = false;
+            Button.SetTitleColor(ThemeHelpers.PrimaryColor, UIControlState.Normal);
+
+            ContentView.Add(Button);
+
+            ContentView.BottomAnchor.ConstraintEqualTo(Button.BottomAnchor, 10f).Active = true;
+
+            switch (buttonsConfig)
+            {
+                case ButtonsConfig.One:
+                    ContentView.AddConstraints(new NSLayoutConstraint[] {
+                        NSLayoutConstraint.Create(ContentView, NSLayoutAttribute.Trailing, NSLayoutRelation.Equal, Button, NSLayoutAttribute.Trailing, 1f, 10f),
+                        NSLayoutConstraint.Create(Button, NSLayoutAttribute.Leading, NSLayoutRelation.Equal, TextField, NSLayoutAttribute.Trailing, 1f, 10f)
+                    });
+                    break;
+                case ButtonsConfig.Two:
+                    SecondButton = new UIButton(UIButtonType.System);
+                    SecondButton.TranslatesAutoresizingMaskIntoConstraints = false;
+                    SecondButton.SetTitleColor(ThemeHelpers.PrimaryColor, UIControlState.Normal);
+
+                    ContentView.Add(SecondButton);
+
+                    ContentView.AddConstraints(new NSLayoutConstraint[] {
+                        NSLayoutConstraint.Create(ContentView, NSLayoutAttribute.Bottom, NSLayoutRelation.Equal, SecondButton, NSLayoutAttribute.Bottom, 1f, 10f),
+                        NSLayoutConstraint.Create(SecondButton, NSLayoutAttribute.Leading, NSLayoutRelation.Equal, TextField, NSLayoutAttribute.Trailing, 1f, 9f),
+                        NSLayoutConstraint.Create(Button, NSLayoutAttribute.Leading, NSLayoutRelation.Equal, SecondButton, NSLayoutAttribute.Trailing, 1f, 10f),
+                        NSLayoutConstraint.Create(ContentView, NSLayoutAttribute.Trailing, NSLayoutRelation.Equal, Button, NSLayoutAttribute.Trailing, 1f, 10f)
+                    });
+                    break;
+            }
+        }
+
+        private float GetTrailingConstant(ButtonsConfig buttonsConfig)
```

Comment:
Even though it's not wrong, I'd improve it by calling it:
`GetTextFieldToContainerTrailingConstant(ButtonsConfig buttonsConfig)`
Given that it's very specific for that case. But will leave that to you if you want to make the change or not